Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -589,13 +589,16 @@ PhysicalParticleContainer::AddPlasmaFromFile(ParticleReal q_tot,
 #if !defined(WARPX_DIM_1D_Z)
         std::shared_ptr<ParticleReal> ptr_x = ps["position"]["x"].loadChunk<ParticleReal>();
         double const position_unit_x = ps["position"]["x"].unitSI();
-#endif
+#endif                                          // 2D
```

Comment:
This is actually 2D, 3D and RZ:
```suggestion
#if !defined(WARPX_DIM_1D_Z)  // 2D, 3D, and RZ
        std::shared_ptr<ParticleReal> ptr_x = ps["position"]["x"].loadChunk<ParticleReal>();
        double const position_unit_x = ps["position"]["x"].unitSI();
#endif
```

Reply:
comment fixed